src/index.js

```diff
@@ -3,8 +3,41 @@
 
 // An example of how you tell webpack to use a CSS (SCSS) file
 import './css/base.scss';
+import goFetch from './fetch-requests.js'
+import User from './User.js'
+import Trip from './Trip.js'
+import Destination from './Destination.js'
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
 import './images/turing-logo.png'
+let user, users, trips, destinations;
+window.addEventListener("load", retrieveData);
+
+function retrieveData() {
+  goFetch.getServerData()
+  .then(responses => Promise.all(responses.map(response => response.json())))
```

Comment:
I like this! It feels a little overcomplicated though. I'm not sure you have to .json the data. It also might be easier to keep that in the fetch request object and then do something with the data afterwords.